src/domain/community/community/community.interface.ts

```diff
@@ -22,7 +23,7 @@ export abstract class ICommunity extends IAuthorizable {
 
   parentCommunity?: ICommunity;
 
-  policy!: string;
+  policy?: ICommunityPolicy;
```

Comment:
why is the policy optional now?

Reply:
Logically it is required, but TypeORM cannot deal with required entity-entity relationship.

Reply:
I don't see any issues with querying the data when it's required.

Reply:
I just made the change...and it is working. I am seriously puzzled....it has been a TypeORM limitation since we started using it that for entity relationships that the fields were optional. Maybe it was only for 1-many and that 1-1 are ok? If so we can probably simplify out quite some 1-1 relationships + undefined checking in the server...

Please also try it + if required works then go ahead + merge; we can raise another card for the systematic review.